bash/runasmdemos

```diff
@@ -1,30 +1,22 @@
 #!/bin/bash
 
 # runasmdemos: run HELLO, DEMOM8Q1, TESTDCB1
-#
-# Note: run from z390/bash directory
-
-# extract longest substring that ends with "/"
-dir=${0%/*}
-
-# get the z390 directory
-zdir=$(dirname $0)
-zdir=$(cd $zdir && pwd)
-zdir=$(dirname $zdir)
 
+set -e              # exit on all errors
+cd $(dirname $0)    # set to directory of script
 cd ..
 
-${dir}/bash/asmlg demo/HELLO $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asmlg demo/HELLO $1 $2 $3 $4 $5 $6 $7 $8 $9
 echo "Verify Hello World WTO"
 
-${dir}/bash/asm demo/DEMOM8Q1 $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm demo/DEMOM8Q1.ZSM $1 $2 $3 $4 $5 $6 $7 $8 $9
```

Comment:
I don't remember DEMOM8Q1.MLC being replaced by DEMOM8Q1.ZSM, but okay, I guess.

Reply:
Oh yeah. So when I ran this script it failed as there was not a .MLC version. I added the prefix so the job would run. Should the file be extension be renamed?

Reply:
I had a discussion about this program some time ago with Don. We then had both an mlc and a zsm version. Since the mlc was generated from the zsm we decided to keep only the latter. If it now needs to be renamed to mlc that's fine with me.